lib/mapper.js

```diff
@@ -24,7 +24,17 @@ exports.identify = function(identify){
   var userId = identify.userId();
   var anonymousId = identify.sessionId();
   var payload = {};
+  var ip = identify.ip();
+  var shouldTrackIp = identify.proxy('context.Kissmetrics.trackIp');
```

Comment:
@f2prateek Which UI are you referring to? The purpose of these flags is to not track IP and user agent for every single request sent to KM. This implementation was inspired by [some of the code in Mixpanel's integration](https://github.com/segmentio/integration-mixpanel/blob/master/lib/index.js#L59-L64).